lab_solutions/lab02_15_pairwise_row_distance.py

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2020 Bence Bial <gbenceg@gmail.com>
+#
+# Distributed under terms of the MIT license.
+
+from lab_solutions.lab02_03_is_matrix import is_matrix
+from lab_solutions.lab02_09_transpose import transpose
+from lab_solutions.lab02_14_l2_disctance import l2_distance
+
+
+def pairwise_row_distance(A):
+    if not is_matrix(A):
+        raise ValueError("Not a valid matrix")
+        
+    l2s = []
+    
+    for i in range(len(A)):
+        l2_row = []
```

Comment:
Filling the matrix with zeros and updating the elements as `A[i][j]` would be more readable.